people-and-planet-ai/geospatial-classification/e2e_test.py

```diff
@@ -0,0 +1,342 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import datetime, timedelta
+import logging
+import os
+import platform
+import subprocess
+import time
+import uuid
+
+import ee
+from google.cloud import aiplatform
+from google.cloud import storage
+import google.auth
+import pandas as pd
+import pytest
+import requests
+
+
+PYTHON_VERSION = "".join(platform.python_version_tuple()[0:2])
+
+NAME = f"ppai/geospatial-classification-py{PYTHON_VERSION}"
+
+UUID = uuid.uuid4().hex[0:6]
+PROJECT = os.environ["GOOGLE_CLOUD_PROJECT"]
+REGION = "us-central1"
+
+TIMEOUT_SEC = 30 * 60  # 30 minutes in seconds
+POLL_INTERVAL_SEC = 60  # 1 minute in seconds
+
+VERTEX_AI_SUCCESS_STATE = "PIPELINE_STATE_SUCCEEDED"
+VERTEX_AI_FINISHED_STATE = {
+    "PIPELINE_STATE_SUCCEEDED",
+    "PIPELINE_STATE_FAILED",
+    "PIPELINE_STATE_CANCELLED",
+}
+
+EARTH_ENGINE_SUCCESS_STATE = "SUCCEEDED"
+EARTH_ENGINE_FINISHED_STATE = {"SUCCEEDED"}
+
+BANDS = [
+    "B1",
+    "B2",
+    "B3",
+    "B4",
+    "B5",
+    "B6",
+    "B7",
+    "B8",
+    "B8A",
+    "B9",
+    "B10",
+    "B11",
+    "B12",
+]
+LABEL = "is_powered_on"
+
+IMAGE_COLLECTION = "COPERNICUS/S2"
+SCALE = 10
+
+TRAIN_VALIDATION_SPLIT = 0.7
+
+PATCH_SIZE = 16
+
+credentials, project = google.auth.default()
```

Comment:
Nit: `project` is no longer needed, can we do `credentials, _ = google.auth.default()`?